Common/Data/Custom/BrainData/BrainDataRankingsFiveDay.cs

```diff
@@ -0,0 +1,95 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Globalization;
+using System.IO;
+
+namespace QuantConnect.Data.Custom.BrainData
+{
+    /// <summary>
+    /// Brain Data Machine Learning Alpha Ranking provides rankings for stocks
+    /// </summary>
+    public class BrainDataRankingsFiveDay : BaseData
+    {
+        /// <summary>
+        /// Ranking data/Machine Learning Alpha score average over period of five days
+        /// </summary>
+        public decimal RankingScore { get; set; }
+
+        /// <summary>
+        /// Empty constructor required for <see cref="Slice.Get{T}()"/>
+        /// </summary>
+        public BrainDataRankingsFiveDay()
+        {
+        }
+
+        /// <summary>
+        /// Gets the source of the data and directs LEAN where to look for it
+        /// </summary>
+        /// <param name="config">Subscription configuration</param>
+        /// <param name="date">Algorithm date</param>
+        /// <param name="isLiveMode">Is live mode</param>
+        /// <returns>Location of the data</returns>
+        public override SubscriptionDataSource GetSource(SubscriptionDataConfig config, DateTime date, bool isLiveMode)
+        {
+            return new SubscriptionDataSource(
+                Path.Combine(
+                    Globals.DataFolder,
+                    "alternative",
+                    "braindata",
+                    "rankings_five_day",
+                    $"{config.Symbol.Value.ToLower()}.csv"
```

Comment:
Are you sure you want to use `Symbol.Value`?

Recall that if the symbol is correctly mapped, `Symbol.Value` is the first part of the `SecurityIdentifier`, meaning the **first** ticker for this security.

Reply:
This behavior is observed in all BrainData `BaseData` implementations.

Reply:
I was under the impression that `Symbol.Value` contained the most recently mapped ticker, not the first ticker. I believe the first ticker can be accessed by `Symbol.ID.Symbol`.

Reply:
You are correct!

Sorry, I was confused.